contents/why_julia.md

```diff
@@ -69,31 +68,261 @@ Let's proceed then!
 
 ## What Julia tries to accomplish? {#sec:julia_accomplish}
 
-Julia [@bezanson2017julia].
+Julia [@bezanson2017julia] is a relatively new language, first released in 2012, aims to be **both easy and fast**.
+It "runs like C^[sometimes even faster than C.] but reads like Python" [@perkelJuliaComeSyntax2019].
+It was made for scientific computing, capable handling **large amounts of data and computation** while still being fairly **easy to manipulate, create and protype code**.
 
-```{=comment}
-My figure from [https://storopoli.io/Bayesian-Julia/pages/images/language_comparisons.svg] (I have it in .drawio and .svg)
-```
+In [@fig:language_comparison] is a highly opinionated representation that divides the main opensource scientific computing languages in a 2x2 diagram with two axes: Slow-Fast and Easy-Hard.
```

Comment:
I explicitly mentioned opensource languages. This is a nasty quirk that I have with MATLAB. We can add it if you really want to. Otherwise, I would like to leave MATLAB out of the discussion.

Reply:
Okay, I've just added it because the text claimed that R was the only academic language :slightly_smiling_face: I'm fine with removing it